packages/models/src/tenant/tenantReadModelAdapter.ts

```diff
@@ -21,13 +23,25 @@ import {
 
 export const toReadModelTenant = (tenant: Tenant): TenantReadModel => ({
   ...tenant,
+  features: tenant.features.map(toReadModelTenantFeature),
   attributes: tenant.attributes.map(toReadModelTenantAttribute),
   createdAt: tenant.createdAt.toISOString(),
   updatedAt: tenant.updatedAt?.toISOString(),
   onboardedAt: tenant.onboardedAt?.toISOString(),
   mails: tenant.mails.map(toReadModelTenantMail),
 });
 
+export const toReadModelTenantFeature = (
+  feature: TenantFeature
+): TenantFeatureReadModel =>
+  match(feature)
+    .with({ type: "PersistentCertifier" }, (feature) => feature)
+    .with({ type: "DelegatedProducer" }, (feature) => ({
+      ...feature,
+      availabilityTimestamp: feature.availabilityTimestamp.toISOString(),
```

Comment:
It's really required to convert this date to ISO string? 
during legacy scala services migration we have conversion like this to guarantee retro compatibility with old Scala implementations, now it's couldn't necessary or I miss something reasons?

Reply:
You're right, there is no need of this. Thanks!